### PR TITLE
🐛 Fix story ads in no signing exp

### PR DESCRIPTION
**Pull request by calebcordry**
In the normal case the cta url & cta type are extracted from the metadata obj and added as data-var attrs to the <amp-ad> element. In the no-signing case the metadata obj does not exist. This PR extends what we query for in the ad doc to find the `<meta name=amp-cta-*>` elements.

Should fix broken experimentA on master builds.

### CODE REVIEW

File: extensions/amp-story-auto-ads/0.1/utils.js [hunk 1]
```diff
@@ -46,11 +46,20 @@ export function getUniqueId(win) {
...
   return doc.querySelectorAll(selector);
 }
 
+/**
+ * Finds all story ad meta tags starting with `amp-cta-`.
+ * @param {Document} doc
+ * @return {!IArrayLike}
+ */
+export function getAmpCtaMetaTags(doc) {
+  return doc.querySelectorAll('meta[name^=amp-cta-]');
```
Comment (lannka) [inline extensions/amp-story-auto-ads/0.1/utils.js:60]: did you try one query to select both?
`meta[name^=amp-cta-],meta[name^=amp4ads-vars-]`

**Review by lannka — COMMENTED** [review]
could you remind me why we had these 2 formats of prefix?

**Comment by calebcordry** [comment]
We started with `amp-cta-url` & `amp-cta-type` but later we thought it might be good to give all these the same `amp4ads-vars` prefix so that we could potentially introduce these to non-story case if needed.

File: extensions/amp-story-auto-ads/0.1/utils.js (hunk 1 above)
Comment (calebcordry) [reply extensions/amp-story-auto-ads/0.1/utils.js:60]: Switched to single selector.

**Review by lannka — APPROVED** [review]

**Pull request merged**